### ISSUE TITLE
Only print report when it has failed build failed on ktlint

### ISSUE THREAD

**Issue by jaredsburrows** [shyiko/ktlint#267]
Feature was added here by this issue: https://github.com/shyiko/ktlint/issues/224.

Example output of `ktlint` as of now:
```
$ gradlew ktlint

> Task :module1:ktlint 
"plain" report written to /some/path

> Task :module2:ktlint 
"plain" report written to /some/path

> Task :module3:ktlint 
"plain" report written to /some/path

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3m 37s
```

I believe we should only print the path if `ktlint` found an issue, especially since it is using `stderr`: https://github.com/shyiko/ktlint/blob/41a5067ad04bfee1ae5975ff774b9076a5bb37c0/ktlint/src/main/kotlin/com/github/shyiko/ktlint/Main.kt#L455.

**Comment by shyiko** [comment]
Hi Jared. Fixed in [0.27.0](https://github.com/shyiko/ktlint/blob/master/CHANGELOG.md#0270---2018-08-06). It should hit Maven Central in about 25 minutes.

**Comment by jaredsburrows** [comment]
@shyiko Thank you.
